### PR TITLE
XP Gain Rate breakdown

### LINKED ISSUE #379: XP Gain Rate not showing Stamina XP %
O percentual de experiência não aparece como deveria, já que indica apenas a experiência base, quando deve indicar também o bônus% de Stamina.

Neste momento, parece assim.

fail

Deveria se parecer com isto.
good

### PR DESCRIPTION
**Pull request by thatmichaelguy**
closes #379 
I can't test at the moment but should be fine, would appreciate if somebody could test.

### CODE REVIEW

**Comment by dudantas** [comment]
Ok. I'll do the tests.
Thanks for the collaboration!

File: data/events/scripts/player.lua [hunk 1]
```diff
@@ -777,7 +777,9 @@ function Player:onGainExperience(source, exp, rawExp)
 	end
 
 	-- Store Bonus
+	local Boost = self:getExpBoostStamina()
 	useStaminaXp(self) -- Use store boost stamina
+	player:setStoreXpBoost(Boost > 0 and 50 or 0)
```
Comment (LukSrT) [inline data/events/scripts/player.lua]: self

Comment (thatmichaelguy) [reply data/events/scripts/player.lua]: Done, thanks

**Comment by majestyotbr** [comment]
Tested and working. Thanks for your contribution.

**Pull request merged**